.github/workflows/build-sdl3.yml

```
@@ -114,7 +114,7 @@ jobs:
         sudo cmake --install . --config Release
 
     - name: Build with SDL3
-      run: python3 dev.py build --sdl3
+      run: python3 dev.py build --sdl3 --ci --commit_hash ${{ github.sha }} --branch "${{ github.head_ref || github.ref_name }}"
```

actionlint (1.7.11)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)